components/p3a/features.cc

```diff
@@ -21,6 +21,9 @@ BASE_FEATURE(kTypicalJSONDeprecation,
 BASE_FEATURE(kOtherJSONDeprecation,
              "BraveP3AOtherJSONDeprecation",
              base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_FEATURE(kNebula,
```

Comment:
this needs short explanation and / or link to a doc

Reply:
I added some doc comments in b89ce25fe962462b59012a0e11f3676084bde330.